Makefile

```diff
@@ -3,6 +3,9 @@
 VERSION ?= 1.9
 export CLOVERAGE_VERSION = 1.0.11-SNAPSHOT
 
+# The test-cljs target needs to be modified if working with JDK9
+JAVA_VERSION = :java$(shell lein version | cut -d " " -f 5 | cut -d "." -f 1-2)
```

Comment:
This is going to produce a string like `:java9.0` (or `:java1.8`, `:java1.7`, it's annoying that the numbering convention changed).

Suggest you simplify by dropping the ":java" part:
```
JAVA_VERSION = $(shell lein version | cut -d " " -f 5 | cut -d "." -f 1-2)
```
which will produce a number like "#.#"